src/shared/types.ts

```diff
@@ -81,9 +81,15 @@ export type AuditListItemEventName =
   | "startAnnualReview"
   | "reminder";
 
+export interface RelatedLink extends PrismaClient.Prisma.JsonObject {
```

Comment:
Out of curiosity, why does this interface extend JsonObject? Will it work without it?

Reply:
sort of, not really. Removing the extend means I have to remove all the extends from the `List` type.. and results in loads of errors elsewhere 😭 

![image](https://github.com/UKForeignOffice/lists/assets/22080510/30bc2ccf-9a08-4c38-ba3d-ac9c809e80d8)

I believe because ts (or our setup of it) doesn't treat a key like a string or objects as serialisable JSON